src/dask_awkward/lib/describe.py

```diff
@@ -19,3 +21,22 @@ def fields(collection: Array | Record) -> list[str] | None:
 
     """
     return collection.fields
+
+
+def backend(*arrays: Array | Record) -> str:
```

Comment:
I don't like this type annotation. Not your fault, but `arrays` is clearly a tuple!

Reply:
I looked this up before writing it ([StackOverflow](https://stackoverflow.com/a/37032111/1623645)). To annotate `*arrays`, we're supposed to specify the type of each `array`. There are ways to unpack this to annotate each field of the tuple differently (it's hard to say how that would help in general, since there's an arbitrary number of them), but since this is a case in which all elements of the `arrays` tuple have the same type, I think this is the simplest correct annotation.